Gruntfile.js

```diff
@@ -70,6 +70,7 @@ module.exports = function( grunt ) {
 					ajax: [ "manipulation/_evalUrl", "event/ajax" ],
 					callbacks: [ "deferred" ],
 					css: [ "effects", "dimensions", "offset" ],
+					"css/showHide": [ "effects" ],
```

Comment:
You're right. I was only thinking of animations pertaining to `.show()` and `.hide()`, not all animations that need to show and hide elements.